mrtarget/plugins/gene/orthologs.py

```diff
@@ -17,12 +17,8 @@ def print_name(self):
 
     def merge_data(self, genes, loader, r_server):
         self._logger.info("Ortholog parsing - requesting from URL %s" % Config.HGNC_ORTHOLOGS)
-        req = requests.get(Config.HGNC_ORTHOLOGS)
-        self._logger.info("Ortholog parsing - response code %s" % req.status_code)
-        req.raise_for_status()
 
-        # io.BytesIO is StringIO.StringIO in python 2
-        for row in csv.DictReader(gzip.GzipFile(fileobj=StringIO(req.content)), delimiter="\t"):
+        for row in csv.DictReader(URLZSource(Config.HGNC_ORTHOLOGS).open()):
```

Comment:
@afaulconbridge the proper way to do this safely is
```
more_itertools.with_iter(URLZSource(filename).open())
```

Reply:
I'm not sure that's required - DictReader should `close()` its source when its exhausted like a normal file. Though it really should be in a "with" statement in case of error yes. I'll push a fix.